dlib/pixel.h

```diff
@@ -771,6 +865,158 @@ namespace dlib
             return(c2);
         }
 
+        // -----------------------------
+
+        struct Lab
+        {
+            double l;
+            double a;
+            double b;
+        };
+        /*
+            Calculate Lab from RGB
+            L is between 0 and 100
```

Comment:
Is this comment correct?  Because code that uses this function seems to assume L has the range -128 to 127.  In any case, the full range of values of l should be used so there isn't loss of dynamic range when using LAB pixels.